resources/backend.rb

```diff
@@ -19,9 +19,7 @@
 # limitations under the License.
 
 resource_name 'chef_backend'
-default_action :create
```

Comment:
What's the reason for these custom resource changes?

Reply:
Defining the default action is a LWRP-ism. We auto set it to the first one in custom resources. You can override that like is happening here, but there's no point.

Reply:
👍